File: Beginner/GCD and LCM (FLOW016)/gcdlcm.c
#include <stdio.h>

int main(void) {
	int i,t;
	scanf("%d",&t);
	for(i=1; i<=t; i++){
	    long long int a, b, x, y, r, gcd, lcm;
	    scanf("%lld%lld", &x, &y);
	    a = x;
	    b = y;
	    while(b!=0) {
	        r = a%b;
	        a = b;
	        b = r;
	    }
	    gcd = a;
	    lcm = (x*y)/gcd;
	    printf("%lld %lld\n",gcd,lcm);
	}
	return 0;
}

File: Beginner/GCD and LCM (FLOW016)/gcdlcm.java
import java.util.Scanner;
class Solution {
	static int gcd(int a, int b) {
		if (a == 0)
			return b; 
		return gcd(b % a, a); 
	}
	
	static int lcm(int a, int b) {
		return (a / gcd(a, b)) * b;
	}
	
	public static void main(String[] args) {
        int a, b, t;
        Scanner scan = new Scanner(System.in);
        t = scan.nextInt(); // input test cases
        for (int i = 0; i < t; ++i) {
            a = scan.nextInt(); // first input integer
            b = scan.nextInt(); // second input integer

            System.out.println(gcd(a, b) + " " + lcm(a, b));
        }
	}
}
